src/Tizen.Multimedia.Util/ImageUtil/ImageEncoder.cs

```diff
@@ -514,13 +514,59 @@ public class WebPEncoder : ImageEncoder
         /// The property determining whether the WebP compression is lossless or lossy.<br/>
         /// The default is false(lossy).</value>
         /// <since_tizen> 8 </since_tizen>
-        public bool Lossless { get; set; }
+        public bool Lossless { get; set; } = false;
 
         internal override void Configure(ImageEncoderHandle handle)
         {
-            NativeEncoder.SetWebPLossless(handle, Lossless).
+            NativeEncoder.SetLossless(handle, Lossless).
                 ThrowIfFailed("Failed to configure encoder; Lossless");
         }
     }
 
+    /// <summary>
+    /// Provides the ability to encode the JPEG(Joint Photographic Experts Group) XL format images.
+    /// </summary>
+    /// <since_tizen> 10 </since_tizen>
+    public class JpegXlEncoder : ImageEncoder
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="JpegXlEncoder"/> class.
+        /// </summary>
+        /// <remarks><see cref="ImageEncoder.OutputFormat"/> will be the <see cref="ImageFormat.JpegXl"/>.</remarks>
+        /// <since_tizen> 10 </since_tizen>
+        public JpegXlEncoder() :
+            base(ImageFormat.JpegXl)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="JpegXlEncoder"/> class with the information for lossless or lossy compression.
```

Comment:
You could consider simplifying the description like this:
```suggestion
        /// Initializes a new instance of the <see cref="JpegXlEncoder"/> class for lossless or lossy compression.
```

Reply:
Fixed.